routing/pathfind.go

```diff
@@ -482,7 +482,7 @@ func findPath(g *graphParams, r *RestrictParams, cfg *PathFindingConfig,
 		var fee lnwire.MilliSatoshi
 		var timeLockDelta uint16
 		if fromVertex != source {
-			fee = computeFee(amountToSend, edge)
```

Comment:
Can `computeFee` be removed now?

Reply:
Yes. Wondering why the linter didn't catch this. I have been experimenting with various settings of `golangci-lint`. It seems that some combinations suppress certain results.